update_data.py

```diff
@@ -1,9 +1,14 @@
 from collections import namedtuple
+from argparse import ArgumentParser
+import calendar
+import datetime
 
 import pandas as pd
+import geopandas
 
 import data_analysis.compare_scheduled_and_rt as csrt
 import data_analysis.plots as plots
```

Comment:
I get an error from this import:

```
Traceback (most recent call last):
  File "/Users/laurie/opt/anaconda3/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/Users/laurie/opt/anaconda3/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/laurie/git/hailey-fork-ghost-buses/update_data.py", line 10, in <module>
    import data_analysis.plots as plots
  File "/Users/laurie/git/hailey-fork-ghost-buses/data_analysis/plots.py", line 34, in <module>
    project_dir = next(p for p in current_dir.parents if p.name == f"{project_name}")
StopIteration
```

Reply:
Noting solution was to manually set the project directory env var but maybe something to document

Reply:
Odd. I don't see this, nor do I have $PROJECT_NAME set in my shell. I added this to the readme though.

Reply:
I think it might be because I downloaded your fork with a different repo name for local testing. Thanks for updating the README 🙏